nav2_smac_planner/lattice_primitives/lattice_generator.py

```diff
@@ -517,6 +533,10 @@ def _create_complete_minimal_spanning_set(
                         )
                     )
 
+                    if (unflipped_trajectory is None or flipped_y_trajectory is None or
```

Comment:
I believe this condition makes the other 2 eq in the `if/else` conditions redundant.

Reply:
Also, is this strictly speaking a problem if there are none-typed trajectories while we're performing this search?

Reply:
@InvincibleRMC can you respond to these items?

Reply:
Sorry I missed this.

> I believe this condition makes the other 2 eq in the `if/else` conditions redundant.

For this do you mean mathematically a path has to exist? Or do you mean it  should be type narrowing somehow?

> Also, is this strictly speaking a problem if there are none-typed trajectories while we're performing this search?

If you prefer it could return an empty Dictionary rather than raising but, it seemed to the output shouldn't be empty.